lib/model/unittest/CResourceMonitorTest.cc

```diff
@@ -325,7 +366,7 @@ BOOST_FIXTURE_TEST_CASE(testMonitor, CTestFixture) {
         std::size_t origTotalMemory = mon.totalMemory();
 
         // Go up to 10 bytes, triggering a need
-        mon.m_CurrentAnomalyDetectorMemory = 10;
+        mon.m_CurrentMonitoredResourceMemory = 10;
```

Comment:
```suggestion
        mon.m_MonitoredResourceCurrentMemory = 10;
```

and elsewhere below...